src/components/10-atoms/toggle-switch/index.scss

```diff
@@ -0,0 +1,60 @@
+$toggle-switch-animation: 0.4s ease;
+$toggle-switch-height: 30px;
+$toggle-switch-width: 50px;
+$toggle-switch-spacing: 2px;
+
+.a-toggle-switch {
+  display: block;
+  position: relative;
+  height: $toggle-switch-height;
+  width: $toggle-switch-width;
+
+  &__input {
+    display: none;
+
+    &:checked {
+      & + .a-toggle-switch__slider {
+        background-color: $color-malachite;
+      }
+
+      & + .a-toggle-switch__slider::before {
+        $distance: $toggle-switch-width - $toggle-switch-height;
+        transform: translateX($distance);
+      }
+
+      &:disabled + .a-toggle-switch__slider {
+        background-color: $color-aqua-green;
+      }
+    }
+
+    &:disabled + .a-toggle-switch__slider {
+      background-color: $color-wild-sand;
+    }
+  } 
+
+  &__slider {
+    position: absolute;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    border-radius: 18px;
+    background-color: $color-light-grey;
+    transition: $toggle-switch-animation;
+    cursor: pointer;
+
+    &::before {
+      $size: $toggle-switch-height - 2 * $toggle-switch-spacing;
```

Comment:
Move $size to the other variables on top of file. (So all the variables can be seen with one view)